vendor/init.te

```diff
@@ -5,12 +5,8 @@ allow init {tmpfs configfs }:lnk_file create_file_perms;
 allow init { mnt_vendor_file vendor_firmware_file adsprpcd_file }:dir mounton;
 allow init vendor_firmware_file:filesystem { mount relabelfrom };
 
-# AOSP init only covers system_block_device, add our own:
-allow init {
-    boot_block_device
-    dtbo_block_device
-    vbmeta_block_device
-}:{ blk_file lnk_file } relabelto;
+# Allow init to relabel boot and vbmeta block devices
```

Comment:
```suggestion
# Allow init to relabel symlinks to boot and vbmeta block devices
```

Reply:
I rewrote the commit message a bit more, hopefully it's better now.

Reply:
@mlehtima I'd suggest rewriting `symlinks dtbo block device` -> `` symlinks to `dtbo_block device` `` for grammar purposes, but other than that this is _exactly_ what I was looking for, thanks!

Feel free to keep the build error in though, that's not harmful and a good demonstration.

Reply:
Done